# Message with all currently tracked persons markers
#

TrackedPersonMarkers[]     tracks      # All persons that are currently being tracked
